pyston/tools/make_release.sh

```diff
@@ -57,9 +58,16 @@ function make_release {
     DIST=$1
 
     echo "Creating $DIST release"
-    docker build -f pyston/Dockerfile.$DIST -t pyston-build:$DIST . --no-cache
-    docker run -iv${PWD}/release/$VERSION:/host-volume --rm --cap-add SYS_ADMIN pyston-build:$DIST sh -s <<EOF
+    docker build -f pyston/make_release_files/Dockerfile.$DIST -t pyston-build:$DIST pyston/make_release_files/ --no-cache
+    docker run -iv${PWD}/release/$VERSION:/host-volume -v${SRC_DIR}:/src/src_dir_host:ro --rm --cap-add SYS_ADMIN pyston-build:$DIST sh -s <<EOF
 set -ex
+
+# make a copy of the source because it's mounted read only and we want to modify it
+cp -r /src/src_dir_host /src/build
```

Comment:
minor improvement but it looks like rsync can take an `--exclude` argument to not copy the build dircetory at all
https://stackoverflow.com/questions/4585929/how-to-use-cp-command-to-exclude-a-specific-directory